src/components/BookState.js

```diff
@@ -0,0 +1,32 @@
+import { books as ArrBooks } from './ListBooks';
+
+function BookState() {
+  return (
+    <section className="d-flex-column books-container">
+      {ArrBooks.map((book) => (
+        <section className="d-flex book-card" key={book.tittle}>
+          <div key={book.tittle}>
+            <h2>{book.tittle}</h2>
+            <p>{book.author}</p>
+            <button type="button">Delete</button>
+          </div>
+          <div className="d-flex-column book-status">
+            <div>
+              <span>{book.completed}</span>
+              <span>Completed</span>
+            </div>
+            <div>
+              <span>Current Chapter</span>
+              <span>
+                Chapter
+                {book.chapter}
+              </span>
+            </div>
+          </div>
+        </section>
```

Comment:
- [ ] The below code snippet should be extracted into a separate component called `Book` or `BookCard` and the book object is passed in as props. Like so 👇🏾  

```javascript
import React from 'react';

const BookCard = ({ book }) => {
  return (
    <section className="d-flex book-card" key={book.title}>
      <div key={book.title}>
        <h2>{book.title}</h2>
        <p>{book.author}</p>
        <button type="button">Delete</button>
      </div>
      <div className="d-flex-column book-status">
        <div>
          <span>{book.completed}</span>
          <span>Completed</span>
        </div>
        <div>
          <span>Current Chapter</span>
          <span>Chapter {book.chapter}</span>
        </div>
      </div>
    </section>
  );
};

export default BookCard;

```

The `BookState` will now consume the `BookCard` like so 👇🏾 

```javascript
import React from 'react';
import BookCard from './BookCard'; // Assuming BookCard component is imported from './BookCard' file
// Import your array of books here

function BookList() {
  
  return (
    <section className="d-flex-column books-container">
      {importedArray.map((book) => (
        <BookCard book={book} key={book.id} /> //You need to add IDs to the book in the books array
      ))}
    </section>
  );
}

export default BookList;
```
By consuming the BookCard component within the BookState component, you can reuse the BookCard component for each book and achieve a modular structure in your React application.